docs/MS Word/Нумерация страниц.md

```diff
@@ -0,0 +1,22 @@
+ В разделе _Нумерация страниц_ выберите _Начать с_, введите нужный номер и нажмите кнопку _ОК_.
+###### Links:
+ * <small>https://lifehacker.ru/kak-vstavit-ubrat-kolontituly-v-word/</small>
+ * <small>https://support.microsoft.com/ru-ru/office/%D0%B2%D1%81%D1%82%D0%B0%D0%B2%D0%BA%D0%B0-%D0%BD%D0%BE%D0%BC%D0%B5%D1%80%D0%BE%D0%B2-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86-9f366518-0500-4b45-903d-987d3827c007#:~:text=%D0%9D%D0%B0%20%D0%B2%D0%BA%D0%BB%D0%B0%D0%B4%D0%BA%D0%B5%20%D0%92%D1%81%D1%82%D0%B0%D0%B2%D0%BA%D0%B0%20%D1%89%D0%B5%D0%BB%D0%BA%D0%BD%D0%B8%D1%82%D0%B5%20%D0%B7%D0%BD%D0%B0%D1%87%D0%BE%D0%BA,%D0%B7%D0%B0%20%D0%B8%D1%81%D0%BA%D0%BB%D1%8E%D1%87%D0%B5%D0%BD%D0%B8%D0%B5%D0%BC%20%D0%BD%D0%B0%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%BD%D1%8B%D1%85%20%D1%82%D0%B8%D1%82%D1%83%D0%BB%D1%8C%D0%BD%D1%8B%D1%85%20%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86.</small>
+ * <small>https://support.microsoft.com/ru-ru/office/%D0%BA%D0%B0%D0%BA-%D0%BD%D0%B0%D1%87%D0%B0%D1%82%D1%8C-%D0%BD%D1%83%D0%BC%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D1%8E-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86-%D0%B2-word-%D0%BD%D0%B5-%D1%81-%D0%BF%D0%B5%D1%80%D0%B2%D0%BE%D0%B9-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%8B-c73e3d55-d722-4bd0-886e-0b0bd0eb3f02#:~:text=%D0%A3%D0%B4%D0%B0%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5%20%D0%BD%D0%BE%D0%BC%D0%B5%D1%80%D0%B0%20%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%8B%20%D1%81%20%D0%BF%D0%B5%D1%80%D0%B2%D0%BE%D0%B9%20%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%8B,-%D0%9D%D0%B0%20%D0%B2%D0%BA%D0%BB%D0%B0%D0%B4%D0%BA%D0%B5%20%D0%92%D1%81%D1%82%D0%B0%D0%B2%D0%BA%D0%B0&text=%D1%80%D0%B0%D0%B7%D0%B4%D0%B5%D0%BB%D0%B0%20%D0%B2%20%D0%BD%D0%B5%D0%BC.-,%D0%92%20%D0%BE%D0%B1%D0%BB%D0%B0%D1%81%D1%82%D0%B8%20%D0%B2%D0%B5%D1%80%D1%85%D0%BD%D0%B5%D0%B3%D0%BE%20%D0%B8%D0%BB%D0%B8%20%D0%BD%D0%B8%D0%B6%D0%BD%D0%B5%D0%B3%D0%BE%20%D0%BA%D0%BE%D0%BB%D0%BE%D0%BD%D1%82%D0%B8%D1%82%D1%83%D0%BB%D0%B0%20%D0%BD%D0%B0%20%D0%BF%D0%B5%D1%80%D0%B2%D0%BE%D0%B9%20%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B5%20%D0%B2%D1%8B,%D0%BE%D0%BA%D0%BD%D0%BE%20%D0%BA%D0%BE%D0%BB%D0%BE%D0%BD%D1%82%D0%B8%D1%82%D1%83%D0%BB%D0%BE%D0%B2%20%D0%B8%D0%BB%D0%B8%20%D0%BA%D0%BB%D0%B0%D0%B2%D0%B8%D1%88%D1%83%20ESC.</small>
+ * <small>https://support.microsoft.com/ru-ru/office/%D0%BA%D0%B0%D0%BA-%D0%BD%D0%B0%D1%87%D0%B0%D1%82%D1%8C-%D0%BD%D1%83%D0%BC%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D1%8E-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86-%D0%B2-word-%D0%BD%D0%B5-%D1%81-%D0%BF%D0%B5%D1%80%D0%B2%D0%BE%D0%B9-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D1%8B-c73e3d55-d722-4bd0-886e-0b0bd0eb3f02#:~:text=%D0%92%D1%8B%D0%B1%D0%B5%D1%80%D0%B8%D1%82%D0%B5%20%D0%92%D1%81%D1%82%D0%B0%D0%B2%D0%BA%D0%B0%20%3E%20%D0%9A%D0%BE%D0%BB%D0%BE%D0%BD%D1%82%D0%B8%D1%82%D1%83%D0%BB%D1%8B.,%2C%20%D0%B2%D1%8B%D0%B1%D0%B5%D1%80%D0%B8%D1%82%D0%B5%20%D0%92%D0%B8%D0%B4%20%3E%20%D0%A0%D0%B5%D0%B6%D0%B8%D0%BC%20%D1%87%D1%82%D0%B5%D0%BD%D0%B8%D1%8F.</small>
```

Comment:
[ch-md-error-eof](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-eof)